.github/workflows/flake_update.yaml

```diff
@@ -0,0 +1,34 @@
+name: "flake_update"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 7:00am on Monday
+    - cron: "0 7 * * 1"
+
+jobs:
+  flake_update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install nix
+        uses: cachix/install-nix-action@v25
+        with:
+          install_url: https://releases.nixos.org/nix/nix-2.20.4/install
+          extra_nix_config: |
+            experimental-features = flakes nix-command
```

Comment:
It looks like this is not needed any more https://github.com/marketplace/actions/install-nix#usage-with-flakes

Reply:
Nice!